src/send/saga.ts

```diff
@@ -7,7 +7,6 @@ import { CeloExchangeEvents, SendEvents } from 'src/analytics/Events'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
 import { ErrorMessages } from 'src/app/ErrorMessages'
 import { calculateFee, currencyToFeeCurrency, FeeInfo } from 'src/fees/saga'
-import { transferGoldTokenLegacy } from 'src/goldToken/actions'
```

Comment:
I believe this whole src/goldToken directory can be deleted as this was the part to use it.